modules/core/shared/src/main/scala/lucuma/core/geom/GmosScienceAreaGeometry.scala

```diff
@@ -16,6 +16,13 @@ import lucuma.core.math.syntax.int._
   */
 object GmosScienceAreaGeometry {
 
+  // base target
+  def base =
+    ShapeExpression.point(Offset.Zero)
```

Comment:
It's trivial but maybe add the `: ShapeExpression` type annotation to these?  I thought that was checked somewhere.